Makefile

```diff
@@ -24,20 +24,18 @@ PERSONDB=FORCE
 ##  The numbers given here RULE
 ##
 
-##  Current series
-SERIES=1.1.1
+##  Current series.  Variable names are numbered to indicate:
+##  SERIES1	OpenSSL pre-3.0
+##  SERIES3	OpenSSL 3.0 and on
+SERIES1=1.1.1
+SERIES3=3.0
```

Comment:
I'd have done `SERIES=$(SERIES1) $(SERIES3)` and left much of the script alone.
There are other changes but this would cover the bulk of them.

Reply:
Okie.  I didn't.

Reply:
... but am changing my mind, having looked again